### ISSUE TITLE
Is it support vllm/sglang inference?

### ISSUE THREAD

**Issue by zshnb** [FireRedTeam/FireRedAudio#3]

**Comment by duduke321** [comment]
A simple vLLM adaptation that only supports audio‑input and text‑output：https://github.com/duduke321/FireRedAudio-VLLM

**Comment by JunjieLl** [comment]
> A simple vLLM adaptation that only supports audio‑input and text‑output：https://github.com/duduke321/FireRedAudio-VLLM

We don’t currently offer official support for vLLM or SGLang, but community integrations and contributions are welcome. Thanks for your work on the vLLM integration! Once it has passed more thorough validation (e.g., logits consistency checks), please feel free to submit a PR.
